src/platform/ESP32/nimble/BLEManagerImpl.cpp

```diff
@@ -827,12 +815,18 @@ void BLEManagerImpl::DriveBLEState(void)
     {
         if (mFlags.Has(Flags::kAdvertising))
         {
+#ifdef CONFIG_BT_NIMBLE_EXT_ADV
+            if (ble_gap_ext_adv_active(kMatterAdvInstance))
+            {
+                err = MapBLEError(ble_gap_ext_adv_stop(kMatterAdvInstance));
+#else
             if (ble_gap_adv_active())
             {
                 err = MapBLEError(ble_gap_adv_stop());
+#endif // CONFIG_BT_NIMBLE_EXT_ADV
                 if (err != CHIP_NO_ERROR)
                 {
-                    ChipLogError(DeviceLayer, "ble_gap_adv_stop() failed: %s", ErrorStr(err));
+                    ChipLogError(DeviceLayer, "BLE advertising stop failed: %s", ErrorStr(err));
```

Comment:
Since we are fixing the log, we can replace `ErrorStr(err)` with `err.Format()`
```suggestion
                    ChipLogError(DeviceLayer, "BLE advertising stop failed: %" CHIP_ERROR_FORMAT, err.Format());
```

Reply:
Done